application_deploy/deploy_demo_apps.sh

```diff
@@ -134,17 +134,18 @@ genExecScript_Func() {
             if [[ -z "${tenantName}" || -z "${namespaceName}" || ${#inputTopicArr[@]} -eq 0 ]]; then
                 echo "[ERROR] Must specify tenant', namespace, and input topic(s) for a Pulsar function."
             else
-                sed -i "s/<tenant_name>/${tenantName}/g" ${funcCfgJsonFileTgt}
-                sed -i "s/<namespace_name>/${namespaceName}/g" ${funcCfgJsonFileTgt}
-                sed -i "s/<func_nam>/${appId}/g" ${funcCfgJsonFileTgt}
+            echo "funcCfgJsonFileTgt is  ${funcCfgJsonFileTgt}"
```

Comment:
Let's remove this debug code, or at least make it this way

```
funcJarFileTgt="${scnHomeDir}/appexec/package/${appId}.jar"
debugMsg "funcJarFileTgt=${funcJarFileTgt}"
```

The debug behavior is controlled by the following setting in `utilities.sh` file
`DEBUG=false`